tests/web_api/test_variables.py

```diff
@@ -94,7 +94,7 @@ def test_variable_formula_github_link():
 
 
 def test_variable_formula_content():
-    formula_code = "def formula(person, period, parameters):\n    return person('salary', period) * parameters(period).taxes.income_tax_rate\n"
+    formula_code = "def formula(person, period, parameters):\n    \"\"\"\n    Income tax.\n\n    The formula to compute the income tax for a given person at a given period\n    \"\"\"\n    return person(\"salary\", period) * parameters(period).taxes.income_tax_rate\n"
```

Comment:
could it be easier to read as:
```suggestion
    formula_code = """
    def formula(person, period, parameters):
        \"\"\"
        Income tax.

        The formula to compute the income tax for a given person at a given period
        \"\"\"
        return person("salary", period) * parameters(period).taxes.income_tax_rate
    """
```

Reply:
^^^ Suggestion only, I'll hit approve button